platform/common/src/main/java/com/tcoded/folialib/impl/ServerImplementation.java

```diff
@@ -266,6 +318,19 @@ public interface ServerImplementation {
      */
     void runAtLocationLater(Location location, @NotNull Consumer<WrappedTask> consumer, long delay, TimeUnit unit);
 
+    /**
+     * Folia: Synced with the tick of the region of the chunk of the location
+     * Paper: Synced with the server main thread
+     * Spigot: Synced with the server main thread
+     * @param world The world to run the task at
+     * @param chunkX The x coordinate of the chunk to run the task at
+     * @param chunkZ The z coordinate of the chunk to run the task at
+     * @param consumer Task to run
+     * @param delay Delay before execution
+     * @param unit Time unit
+     */
+    void runAtLocationLater(World world, int chunkX, int chunkZ, @NotNull Consumer<WrappedTask> consumer, long delay, TimeUnit unit);
```

Comment:
This should return a `CompleteableFuture<Void>`, matching the other methods from #16